src/spikeinterface/comparison/comparisontools.py

```diff
@@ -232,7 +176,61 @@ def compute_matching_matrix(
     return compute_matching_matrix
 
 
-def make_match_count_matrix(sorting1, sorting2, delta_frames):
+def make_match_count_matrix(sorting1, sorting2, delta_frames, symmetric=False):
+    """
+    Compute a matrix representing the matches between two Sorting objects.
+
+    Given two spike trains, this function finds matching spikes based on a temporal proximity criterion
+    defined by `delta_frames`. The resulting matrix indicates the number of matches between units
+    in `spike_frames_train1` and `spike_frames_train2` for each pair of units.
+
+    Note that this algo is not symmetric and biased toward sorting1 is the ground truth.
```

Comment:
```suggestion
    Note that this algo is not symmetric and is biased with `sorting1` representing ground truth for the comparison
```

Just trying to clean this a bit. But maybe I made it too strong?